### PR TITLE
closes #27 Use go-shellwords when parse task commands

### LINKED ISSUE #27: A command containing quotes can not be split properly
In ECS, we can inject environment variables from the SSM parameter store by the docker ENTRYPOINT. So, I want to delay evaluate the environment variables in command arguments. 

```
$ ecs-goploy  --command='/bin/sh -c "echo $DATABASE_URL"'
```

However, the current implementation simply splits command by space.

https://github.com/crowdworks/ecs-goploy/blob/e3326d2706411e598db5be91a42a2c1a192c777d/deploy/task.go#L63

It would be great if the command can be split the string containing quotes properly.

### PR DESCRIPTION
**Pull request by h3poteto**
For example, `bash -c "echo $PORT"` will return `["bash", "-c", "echo $PORT"]` using go-shellwords.

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 [[projects]]
   name = "github.com/aws/aws-sdk-go"
-  packages = ["aws","aws/awserr","aws/awsutil","aws/client","aws/client/metadata","aws/corehandlers","aws/credentials","aws/credentials/ec2rolecreds","aws/credentials/endpointcreds","aws/credentials/stscreds","aws/defaults","aws/ec2metadata","aws/endpoints","aws/request","aws/session","aws/signer/v4","internal/shareddefaults","private/protocol","private/protocol/json/jsonutil","private/protocol/jsonrpc","private/protocol/query","private/protocol/query/queryutil","private/protocol/rest","private/protocol/xml/xmlutil","service/ecs","service/sts"]
+  packages = ["aws","aws/awserr","aws/awsutil","aws/client","aws/client/metadata","aws/corehandlers","aws/credentials","aws/credentials/ec2rolecreds","aws/credentials/endpointcreds","aws/credentials/stscreds","aws/defaults","aws/ec2metadata","aws/endpoints","aws/request","aws/session","aws/signer/v4","internal/shareddefaults","private/protocol","private/protocol/json/jsonutil","private/protocol/jsonrpc","private/protocol/query","private/protocol/query/queryutil","private/protocol/rest","private/protocol/xml/xmlutil","service/ecs","service/ecs/ecsiface","service/sts"]
```
Comment (minamijoyo) [inline Gopkg.lock]: What is this change?

File: deploy/task.go [hunk 2]
```diff
@@ -60,7 +60,11 @@ func NewTask(cluster, name, imageWithTag, command string, baseTaskDefinition *st
 			*tag,
 		}
 	}
-	commands := strings.Split(command, " ")
+	p := shellwords.NewParser()
+	commands, err := p.Parse(command)
+	if err != nil {
+		return nil, errors.Wrap(err, "Parse error in a task command")
+	}
```
Comment (minamijoyo) [inline deploy/task.go]: 👍

File: Gopkg.lock (hunk 1 above)
Comment (h3poteto) [reply Gopkg.lock]: It is result of `dep ensure`. I ran the command when I add go-shellwords.

Comment (minamijoyo) [reply Gopkg.lock]: ok

**Comment by minamijoyo** [comment]
I've checked this branch and it works fine for me. Thanks!

**Pull request merged**